### ISSUE TITLE
Show non-DOT Treasury Assets

### ISSUE THREAD

**Issue by joepetrowski** [opensquare-network/dotreasury#1381]
dotreasury only shows the DOT in the Treasury, it should also show other assets like USDT/USDC on its [account](https://assethub-polkadot.subscan.io/account/14xmwinmCEz6oRrFdczHKqHgWNMiCysE2KrA4jXXAAM1Eogk)

![image](https://github.com/opensquare-network/dotreasury/assets/25483142/e8083748-29a0-4e27-a8bb-9fc0be7015ad)

**Comment by wliyongfeng** [comment]
Hey @joepetrowski , thanks for this issue. We are designing new components for assets on asset hub.

**Comment by Popoulosss** [comment]
@wliyongfeng 

- Add a panel to show the available treasury balance and assets of `Assethub` to the overview page
  - Add a assets pie chart
  - click the assets symbol to link to the Statescan
     - [USDC](https://statemint.statescan.io/#/assets/1337)
     - [USDt](https://statemint.statescan.io/#/assets/1984)
- Add titles to distinguish the panels of `Relay Chain` and `Assethub`, click the title to link to the SubScan
  - [relay chain account](https://polkadot.subscan.io/account/13UVJyLnbVp9RBZYFwFGyDvVd1y27Tt8tkntv6Q7JVPhFsTB)
  - [assethub account](https://assethub-polkadot.subscan.io/account/14xmwinmCEz6oRrFdczHKqHgWNMiCysE2KrA4jXXAAM1Eogk)

Files
[icons.zip](https://github.com/user-attachments/files/15864642/icons.zip)

Figma link
<img width="640" alt="image" src="https://github.com/opensquare-network/dotreasury/assets/59445675/7dfc1eb1-924d-4ad9-aa37-8930c05cd859">

**Comment by joepetrowski** [comment]
> Add titles to distinguish the panels of Relay Chain and Assethub, click the title to link to the SubScan

Personally I would strongly advocate NOT to do this. The Treasury is a multi-chain app. I don't see any reason to expose the underlying chain architecture to the user. If you are cloud-hosting a website, you don't show which data is being served from which region, you just show the data.

It should show the Treasury as a single, unified thing.

**Comment by wliyongfeng** [comment]
> It should show the Treasury as a single, unified thing.

@joepetrowski Or we add another panel to show the unified value with all chains value(relay chain, assethub, and maybe collectives later). I think there are scenarios users want to know treasury value distribution by different chains, so it's not a bad thing to show detailed values on different chains IMO.

**Comment by joepetrowski** [comment]
> it's not a bad thing to show detailed values on different chains IMO.

It is a bad thing IMO because it exposes architecture to a user. IMO most users should not even know that things like Asset Hub, Collectives, People, etc. exist. The entire point of Polkadot/shared security was that we can push specialized logic/state in separate places but still have them interact "as one". The _entire thing_ should just be "Polkadot".

When you order from Amazon do they show you their database schema? Do they tell you which items are stored in which warehouse? No, because they user doesn't care and doesn't need to know. The user wants to use Amazon, not learn about their application backend and distribution network.

If you must have the separation, it should be hidden in an "advanced"/"dev" tab, because sure some developers will need to know about this architecture when working on their apps (for example, knowing where to submit a transaction). But the users using those applications should just use "Polkadot". The unified view has to be the default.

**Comment by Popoulosss** [comment]
Preview
<img width="640" alt="image" src="https://github.com/opensquare-network/dotreasury/assets/59445675/875de7c0-cbd2-4280-b290-f58cb4b9cb2b">

**Comment by joepetrowski** [comment]
Yeah that is looking better! Still a few suggestions about organization:

1. I would make this pie chart with asset values include the DOT on the Relay Chain and make it more prominent. And then I would have a "Details" tab _just on that_ that shows what is on the Relay Chain and what is on Asset Hub.

![image](https://github.com/opensquare-network/dotreasury/assets/25483142/0ce64348-9eab-47c6-9e6e-fd59fa23bfce)

2. I would NOT link the following info with Relay Chain. The reason is, it has nothing to do with the assets. It's just not true that "22.99 M DOT is available on the Relay Chain", because in fact ~28M DOT and 6M USDT and 6M USDC are available. Key point: **These are not separate treasuries, it is one treasury**. The Relay Chain happens to be where the decision making logic is that controls the spending, and it also happens to be where some of the assets are stored. But we could have a Treasury parachain that just has spending logic/governance tracks and it could have assets on 20 different parachains. These things (assets and spending logic) need to be seen as separate components of one system. The fact that the decision making logic is on the Relay Chain is irrelevant to the information.

![image](https://github.com/opensquare-network/dotreasury/assets/25483142/84670852-9230-4f80-98ce-12ac41a501c5)

What I am getting at is to think a few steps ahead. After [Ref 832](https://polkadot.polkassembly.io/referenda/832) passes, the Fellowship Treasury will have its decision making logic on the Collectives chain and assets on Asset Hub. It just doesn't make sense to split them up, it is one system and we should not expose this architecture to end-users.

**Comment by Popoulosss** [comment]
@joepetrowski Thank you very much for your suggestions. We will discuss and then redesign the current version based on your feedback. We are also developing a prototype of doTreasury2, which will better showcase assets from the Relay chain, Assethub, and sub treasury in one system(one treasury).
